openstf/dataclasses/model_specifications.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: 2021 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+from pydantic import BaseModel
+from typing import Optional, Union
+
+
+class ModelSpecificationDataClass(BaseModel):
```

Comment:
Do we want ModelSpecificationDataClass defined twice (once in openstf and once in openstf-dbc?)? We will very likely run into issues in the future, because datclass versions are different. Plus it will be harder to maintain. So I think it would be wiser to import the class from openstf-dbc and remove this code?

Reply:
Will be removed from openstf-dbc in the future